src/Nethermind/Nethermind.Evm/TransactionSubstate.cs

```diff
@@ -47,24 +47,32 @@ public TransactionSubstate(EvmExceptionType exceptionType, bool isTracerConnecte
                 {
                     if (Output.Length > 0)
                     {
-                        try
-                        {
-                            BigInteger start = Output.Span.Slice(4, 32).ToUnsignedBigInteger();
-                            BigInteger length = Output.Slice((int)start + 4, 32).Span.ToUnsignedBigInteger();
-                            Error = string.Concat("Reverted ",
-                                Output.Slice((int)start + 32 + 4, (int)length).ToArray().ToHexString(true));
-                        }
-                        catch (Exception)
+                        var span = Output.Span;
+                        if (span.Length > 32 + 4)
```

Comment:
This was not there before. Why? And what does this have to do with JSON? Can't you put it on a different PR?

Reply:
Was annoying me when debugging. Have added separate PR https://github.com/NethermindEth/nethermind/pull/5727